### ISSUE TITLE
[bug] Slider inside Dialog with step < 1 issues wrong values on some steps

### ISSUE THREAD

**Issue by apasov** [quasarframework/quasar#1121]
### Software version

Quasar: 0.4.17
OS: Windows 10
Node: 9.1.0
Browsers: Chrome

### What did you get as the error?
At some steps values are e.g. 0.41000000000000003 instead of 0.41
![image](https://user-images.githubusercontent.com/28765662/32701561-ce674c82-c7e0-11e7-8564-9ec3a437bad6.png)
At other steps everything's okay.
![image](https://user-images.githubusercontent.com/28765662/32701573-fdf17d92-c7e0-11e7-8abd-0180b0b2bdcf.png)

### What steps did you take, to get the error?

Here's the code of the Slider as part of a Dialog:
```javascript
handler: () => {
  Dialog.create({
    title: 'Dialog',
    form: {
      slider: {
        type: 'slider',
        model: 0.3,
        label: 'Slider',
        min: 0.01,
        max: 1.0,
        step: 0.01,
        withLabel: true
      }
    }
  })
}
```

**Comment by rstoenescu** [comment]
Will take care of this. Thanks for reporting!

**Comment by rstoenescu** [comment]
Available in edge and future v0.14.8
